### ISSUE TITLE
how to install dev-tools electron-app: readme request

### ISSUE THREAD

**Issue by smilingkite** [vuejs/vue-devtools#1107]
### What problem does this feature solve?
I don't understand a line in the readme - when I do, I should be able to run the dev-tools to work with IE. 

### What does the proposed API look like?
The line I don't understand reads:
run: 

`./node_modules/.bin/vue-devtools`

I tried copy pasting that into my console, but it doesn't work.
I tried 

`npm run ./node_modules/.bin/vue-devtools`

Which also doesn't work. 

So HOW to run it would be nice to have in the readme at: 
https://github.com/vuejs/vue-devtools/blob/dev/packages/shell-electron/README.md

**Comment by smilingkite** [comment]
I created a pull request for this one:

**Comment by smilingkite** [comment]
https://github.com/vuejs/vue-devtools/pull/1108

**Comment by LinusBorg** [comment]
> I tried copy pasting that into my console, but it doesn't work.

... but that's how it works. an you provide a little more insight as to what you did exactly, on which OS, which terminal? And what the result was?

**Comment by smilingkite** [comment]
I went the 'Using dependency package' route, so I didn't install the package globally. I did install it into my local project, and ran the command within the app. In the terminal as provided by WebStorm. and nothing happened. It didn't recognize it as a command. 

The following procedure did work: 

http://www.hesselinkwebdesign.nl/2019/vue-dev-tools-for-development-in-ie-11/

I understand writing readme's is hard, but I do feel the readme is missing a lot of 'but of course' information that newby to electron just doesn't know.
